lib/events.js

```diff
@@ -394,6 +394,27 @@ EventEmitter.prototype.listeners = function listeners(type) {
   return ret;
 };
 
+EventEmitter.prototype.listenerCount = function listenerCount(type) {
+
+  if (typeof type !== 'string') {
+    throw new TypeError('type must be a string');
+  }
+
+  const events = this._events;
+
+  if (events) {
+    const evlistener = events[type];
+
+    if (typeof evlistener === 'function') {
+      return 1;
+    } else if (evlistener) {
+      return evlistener.length;
+    }
+  }
+
+  return 0;
+};
```

Comment:
We shouldn't enforce string for the type. It would not be possible to count events using a Symbol.
Furthermore, if we eventually switch to a Map implementation, any type could be used.